--- a/setup.sh	2023-01-24 22:34:24.326093638 +0800
+++ b/setup.sh	2023-01-24 23:07:17.042880877 +0800
@@ -77,6 +77,7 @@
 audit_store=$(get_prop 'audit_store' $PROPFILE)
 audit_elasticsearch_urls=$(get_prop 'audit_elasticsearch_urls' $PROPFILE)
 audit_elasticsearch_port=$(get_prop 'audit_elasticsearch_port' $PROPFILE)
+audit_elasticsearch_protocol=$(get_prop 'audit_elasticsearch_protocol' $PROPFILE)
 audit_elasticsearch_user=$(get_prop 'audit_elasticsearch_user' $PROPFILE)
 audit_elasticsearch_password=$(get_prop 'audit_elasticsearch_password' $PROPFILE)
 audit_elasticsearch_index=$(get_prop 'audit_elasticsearch_index' $PROPFILE)
@@ -794,6 +795,10 @@
 		newPropertyValue=${audit_elasticsearch_port}
 		updatePropertyToFilePy $propertyName $newPropertyValue $to_file_ranger
 
+		propertyName=ranger.audit.elasticsearch.protocol
+		newPropertyValue=${audit_elasticsearch_protocol}
+		updatePropertyToFilePy $propertyName $newPropertyValue $to_file_ranger
+
 		propertyName=ranger.audit.elasticsearch.user
 		newPropertyValue=${audit_elasticsearch_user}
 		updatePropertyToFilePy $propertyName $newPropertyValue $to_file_ranger


--- a/ews/webapp/WEB-INF/classes/conf.dist/ranger-admin-site.xml	2023-01-24 14:56:45.061000000 +0000
+++ b/ews/webapp/WEB-INF/classes/conf.dist/ranger-admin-site.xml	2023-01-24 14:57:05.978000000 +0000
@@ -56,6 +56,11 @@
 		<description></description>
 	</property>
 	<property>
+		<name>ranger.audit.elasticsearch.protocol</name>
+		<value>http</value>
+		<description></description>
+	</property>
+	<property>
 		<name>ranger.audit.elasticsearch.user</name>
 		<value></value>
 		<description></description>

